test/run-e2e.sh

```diff
@@ -28,6 +28,7 @@ cleanup() {
 	# shell check  ignore word splitting when using jobs -p
 	# shellcheck disable=SC2046
 	[[ -z "$(jobs -p)" ]] || kill $(jobs -p) || true
+	return 1
```

Comment:
we should return 1 here as well.

Reply:
If we don't return 1 here, `run-e2e-ocp.sh` calls `run-e2e.sh` and suppose `run-e2e.sh` end normally and continue it's run and execute clean, the trap invoke clean function also